molecule/_shared/base.yml

```diff
@@ -21,6 +21,7 @@ lint: |
   yamllint .
   ansible-lint
   flake8
+role_name_check: 1
```

Comment:
Yes, see:

https://molecule.readthedocs.io/en/latest/configuration/#role-name-check

Unfortunately it is necessary because this role's name doesn't comply with the new Ansible naming standards. At some point in time, it would be nice if we could rename the role, but that's a much larger discussion and probably also one that should be taken up with other elements of the Ansible community.

Reply:
Ever since moving this into this organization it couldn't be published to Galaxy. Hashicorp doesn't want to spend resources on this. Maybe the namespace in meta/main.yml can be changed to comply. You could maybe setup another organization?

Reply:
Right, yes, I recall that now. I am actually happy with the role being under the ansible-community org, and moving it might be disruptive for users. I'll have a look at what could be done for the namespace. Again, this is part of a bigger discussion, and if I find a solution it'll be fixed in a separate PR.